postcss.config.js

```diff
@@ -3,7 +3,16 @@ const path = require( 'path' );
 module.exports = ( { options: { preserveCssCustomProperties = true } } ) => ( {
 	plugins: {
 		'postcss-custom-properties': {
-			importFrom: [ path.join( __dirname, 'public', 'custom-properties.css' ) ],
+			importFrom: [
+				path.join(
+					__dirname,
+					'node_modules',
+					'@automattic',
+					'calypso-css-custom-properties',
```

Comment:
i think you could get up to this point with `require.resolve( '@automattic/calypso-css-custom-properties' )`? maybe?

Reply:
Yep, `require.resolve` will return the whole desired path, including the `dist/custom-properties.css` bit. It will look at the `main` field in `package.json`, too. To resolve the index file.

Reply:
Much nicer, thanks both! d7a68d21ad